### ISSUE TITLE
Create the first release Wx-AQ_v1.0

### ISSUE THREAD

**Issue by weiwilliam** [weiwilliam/Wx-AQ_LuLab#34]
Once PR #33 is merged into the master branch.
We can create the first release of Wx-AQ system Wx-AQ_v1.0.

**Comment by weiwilliam** [comment]
tag of Wx-AQ v1.0.0 is released.
